codespell_lib/data/dictionary.txt

```diff
@@ -5098,6 +5117,7 @@ discourrage->discourage
 discourraged->discouraged
 discovereability->discoverability
 discovr->discover
+discreet->discrete, discreet,
```

Comment:
will return false positives but since this is a common misspelling LGTM

Reply:
Agreed.

Reply:
I don't know what @larsoner thinks, but personally I feel there shouldn't be valid spellings in the primary dictionary. To take a not hugely contrived example, `passwords must be stored in a discreet way with each one in a discrete file`.

Reply:
This correction should now be in rare if it goes in at all:
https://github.com/codespell-project/codespell/blob/master/codespell_lib/data/dictionary_rare.txt